akvo/templates/project_directory.html

```diff
@@ -78,6 +78,13 @@
       {% blocktrans with start_index=page.start_index end_index=page.end_index paginator_count=paginator.count %}
           Viewing {{ start_index }} - {{ end_index }} of {{ paginator_count }} projects
       {% endblocktrans %}
+          | View
+          {% with "20 50 100" as page_sizes %}
```

Comment:
Two issues here. 

I think the settings.PROJECT_DIRECTORY_DEFAULT_SIZE page size should be included in the list.

When changing page size your "cursor" is eliminated. By this I mean that the page counter is kept the same but the page size changes so you unless you're on the first page you will see a totally different set of projects. Ideally you'd have the same first project when you switch page size but I can see that might be hard to do given the rather blunt page=X&limit=Y, but maybe it's worth trying to calculate the page so that the first project on the old page is part of the new page.